frontend/public/components/pod-logs.jsx

```diff
@@ -3,16 +3,37 @@ import * as React from 'react';
 
 import { Dropdown, ResourceLog, ResourceName } from './utils';
 
-// Component to container dropdown or conatiner name if only one container in pod.
-const ContainerDropdown = ({currentContainer, containers, kind, onChange}) => {
-  const resourceName = (container) => {
-    return <ResourceName name={container.name} kind={kind} />;
+const reduceContainerStatuses = (accumulator, container, order) => {
+  return {
+    ...accumulator,
+    [container.name]: { ...container, order }
   };
-  const dropdownItems = _.mapValues(containers, resourceName);
+};
+
+const containerLabel = (container) => {
+  return <ResourceName name={container.name} kind="Container" />;
+};
+
+const ContainerDropdown = ({currentKey, containers, initContainers = {}, onChange}) => {
+  let spacerBefore = new Set();
+  let headerBefore = {};
+  if (!_.isEmpty(initContainers)) {
+    const firstInitContainer = _.find(initContainers, {order: 0});
+    const firstAppContainer = _.find(containers, { order: 0 });
```

Comment:
nit: use consistent space before / after curly braces

Reply:
Bonus to fix with another lint rule 😄